libraries/EmbeddedChessPieces/src/ECPChessLogic/ECPChessPieces/ECPPawn.cpp

```diff
@@ -0,0 +1,52 @@
+#include "ECPPawn.h"
+
+ECPPawn::ECPPawn(Dezibot &d, ECPChessField initialField, bool isWhite)
+    : ECPChessPiece(d, initialField, isWhite) {};
+
+bool ECPPawn::isMoveValid(ECPChessField newField) {
+    const bool doesNotChangeColumn = currentField.column == newField.column;
```

Comment:
For consistency:
```suggestion
    if (newField == currentField) {
        // pawn must move
        return false;
    }
    
    const bool doesNotChangeColumn = currentField.column == newField.column;
```

Reply:
This is actually already covered in the implementation. Shall I add it anyways to stay consistent?

Reply:
That's what I meant with 'for consistency', since it is used in the implementation of the knight as well, even when it's covered.

Reply:
Added in 564708358e82b65942cae862695dfb3485382f6b